src/Microsoft.ML.Data/Utils/LossFunctions.cs

```diff
@@ -222,19 +262,46 @@ public float DualUpdate(float output, float label, float alpha, float invariant,
             return maxNumThreads >= 2 && Math.Abs(fullUpdate) > Threshold ? fullUpdate / maxNumThreads : fullUpdate;
         }
 
-        public Double DualLoss(float label, float dual)
+        public double DualLoss(float label, float dual)
         {
             if (label <= 0)
                 dual = -dual;
 
             // The dual variable is out of the feasible region [0, 1].
             if (dual < 0 || dual > 1)
-                return Double.NegativeInfinity;
+                return double.NegativeInfinity;
 
             return _margin * dual;
         }
     }
 
+    /// <summary>
+    /// A smooth version of the Hinge Loss function, commonly used in classification tasks.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// Let $L(f(\vec{x}, y) = 1 - yf(\vec{x})$, where $f(\vec{x})$ is the predicted score and $y \in \{-1, 1\}$ is the true label.
+    ///
+    /// Note that the labels used in this calculation are -1 and 1, unlike Log Loss, where the labels used are 0 and 1.
+    /// Also unlike Log Loss, $f(\vec{x})$ is the raw predicted score, not the predicted probability (which is calculated by applying a [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to the predicted score).
+    ///
+    /// The Smoothed Hinge Loss function is then defined as:
+    ///
+    /// \[
+    /// g(L(f(\vec{ x}, y)) =
+    /// \begin{cases}
```

Comment:
@natke @sfilipi do you know if this will render? This is using the amsmath LaTeX package.

Reply:
Preview looks good: https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.trainers.logloss?branch=smoke-test-preview&view=ml-dotnet

Reply:
This doesn't work. For a start, there is no $ $ surrounding the latex. You can test it by editing the smoke-test-preview branch here:  https://github.com/dotnet/ml-api-docs/blob/smoke-test-preview/dotnet/xml/Microsoft.ML.Trainers/SmoothedHingeLoss.xml

Reply:
The \\[ and \\] should have worked the same as $ $. In any case, I've updated this PR to use $ $ instead.

I tried pushing a commit to the smoke-test-preview branch but got access denied.